renovate.json5

```diff
@@ -1,10 +1,51 @@
 {
   $schema: "https://docs.renovatebot.com/renovate-schema.json",
   extends: [
-    "config:base",
+    "config:base"
   ],
-  ignorePaths: [
-      "src/jvmTest/resources/",
+  enabled: true,
+  enabledManagers: [
```

Comment:
Why does it make sense to limit what package managers are enabled? I usually rely on the default.

Reply:
I've not got a strong reason. It's just for explicitness, and to be defensive.